rewatch/src/build.rs

```diff
@@ -241,15 +237,19 @@ pub fn initialize_build(
     let timing_cleanup_elapsed = timing_cleanup.elapsed();
 
     if show_progress {
-        println!(
-            "{}{} {}Cleaned {}/{} {:.2}s",
-            LINE_CLEAR,
-            style("[4/7]").bold().dim(),
-            SWEEP,
-            diff_cleanup,
-            total_cleanup,
-            default_timing.unwrap_or(timing_cleanup_elapsed).as_secs_f64()
-        );
+        if snapshot_output {
+            println!("Cleant {}/{}", diff_cleanup, total_cleanup)
```

Comment:
I don't think "cleant" is a thing 🙂 (occurs in several places).

```suggestion
            println!("Cleaned {}/{}", diff_cleanup, total_cleanup)
```

Reply:
Yes my bad, will fix this before merge.